.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+repos:
+  - repo: https://github.com/asottile/reorder_python_imports
+    rev: v1.2.0
+    hooks:
+      - id: reorder-python-imports
+        args: ["--application-directories", "src"]
+  - repo: https://github.com/ambv/black
```

Comment:
not a fan of black due to their idiotic decision of using double quotes everywhere while the majority of python projects/developers use/prefer single quotes

Reply:
I've been using it in a bunch of projects and at this point it doesn't bother me. But the thing that made it click was that you can continue to write your code in whatever style you want, then let Black standardize it.

Reply:
Which is super ugly because then you write it in one style while having to read it in another style